<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1116</width>
    <height>688</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>640</width>
          <height>320</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>                                                                                             Add navigation point</string>
          </property>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>316</width>
             <height>301</height>
            </rect>
           </property>
           <property name="title">
            <string>Virtual stick</string>
           </property>
           <widget class="QPushButton" name="enableVirtualStickButton">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>141</width>
              <height>25</height>
             </rect>
            </property>
            <property name="text">
             <string>Enable virtual Stick</string>
            </property>
           </widget>
           <widget class="QSlider" name="yawSlider">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>100</y>
              <width>121</width>
              <height>16</height>
             </rect>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
           <widget class="QSlider" name="pitchSlider">
            <property name="geometry">
             <rect>
              <x>169</x>
              <y>100</y>
              <width>131</width>
              <height>20</height>
             </rect>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
           <widget class="QSlider" name="rollSlider">
            <property name="geometry">
             <rect>
              <x>210</x>
              <y>130</y>
              <width>16</width>
              <height>121</height>
             </rect>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
           <widget class="QLabel" name="yawLabel">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>80</y>
              <width>67</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Yaw</string>
            </property>
           </widget>
           <widget class="QLabel" name="rollLabel">
            <property name="geometry">
             <rect>
              <x>230</x>
              <y>170</y>
              <width>31</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Roll</string>
            </property>
           </widget>
           <widget class="QPushButton" name="onRestStickValueButton">
            <property name="geometry">
             <rect>
              <x>100</x>
              <y>260</y>
              <width>89</width>
              <height>25</height>
             </rect>
            </property>
            <property name="text">
             <string>reset</string>
            </property>
           </widget>
           <widget class="QLabel" name="pitchLabel">
            <property name="geometry">
             <rect>
              <x>190</x>
              <y>80</y>
              <width>67</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Pitch</string>
            </property>
           </widget>
           <widget class="QPushButton" name="disableVirtualStickButton">
            <property name="geometry">
             <rect>
              <x>170</x>
              <y>30</y>
              <width>131</width>
              <height>25</height>
             </rect>
            </property>
            <property name="text">
             <string>Disable virtual S</string>
            </property>
           </widget>
           <widget class="QSlider" name="throttleSlider">
            <property name="geometry">
             <rect>
              <x>60</x>
              <y>130</y>
              <width>16</width>
              <height>121</height>
             </rect>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
           <widget class="QLabel" name="throttleLabel">
            <property name="geometry">
             <rect>
              <x>80</x>
              <y>170</y>
              <width>67</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Throttle</string>
            </property>
           </widget>
           <widget class="QPushButton" name="setRollButton">
            <property name="geometry">
             <rect>
              <x>230</x>
              <y>190</y>
              <width>61</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>Set</string>
            </property>
           </widget>
           <widget class="QLabel" name="yawBiasLabel">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>60</y>
              <width>67</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="pitchBiasLabel">
            <property name="geometry">
             <rect>
              <x>190</x>
              <y>60</y>
              <width>67</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="throttleBiasLabel">
            <property name="geometry">
             <rect>
              <x>80</x>
              <y>150</y>
              <width>67</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="rollBiasLabel">
            <property name="geometry">
             <rect>
              <x>230</x>
              <y>150</y>
              <width>67</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
          <widget class="QSlider" name="zoomSlider">
           <property name="geometry">
            <rect>
             <x>430</x>
             <y>40</y>
             <width>160</width>
             <height>16</height>
            </rect>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
          <widget class="QLabel" name="label_7">
           <property name="geometry">
            <rect>
             <x>350</x>
             <y>40</y>
             <width>67</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Zoom</string>
           </property>
          </widget>
          <widget class="QPushButton" name="goStraightButton">
           <property name="geometry">
            <rect>
             <x>430</x>
             <y>240</y>
             <width>89</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Go straight</string>
           </property>
          </widget>
          <widget class="QPushButton" name="turnLeftButton">
           <property name="geometry">
            <rect>
             <x>330</x>
             <y>240</y>
             <width>91</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Turn left</string>
           </property>
          </widget>
          <widget class="QPushButton" name="turnRightButton">
           <property name="geometry">
            <rect>
             <x>530</x>
             <y>240</y>
             <width>89</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Turn right</string>
           </property>
          </widget>
          <widget class="QLabel" name="turnLabel">
           <property name="geometry">
            <rect>
             <x>420</x>
             <y>180</y>
             <width>111</width>
             <height>51</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QCheckBox" name="manulDirectCheckBox">
           <property name="geometry">
            <rect>
             <x>330</x>
             <y>270</y>
             <width>151</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Manul direct</string>
           </property>
          </widget>
          <widget class="QSlider" name="navPointSlider">
           <property name="geometry">
            <rect>
             <x>430</x>
             <y>90</y>
             <width>160</width>
             <height>16</height>
            </rect>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
          <widget class="QLabel" name="label_9">
           <property name="geometry">
            <rect>
             <x>350</x>
             <y>90</y>
             <width>67</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Nav point</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="minimumSize">
       <size>
        <width>450</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Map</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWebEngineView" name="webView" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>300</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Velocity_V:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="VelV">
             <property name="text">
              <string>3.3</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>m/s</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Velocity_H:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="VelH">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>3.3</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>m/s</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Battery:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Bat">
             <property name="text">
              <string>99</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>%</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>GPS Data</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Map_Refresh</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Lat</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditLat">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>29.6169</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Lng</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditLng">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>106.4988</string>
            </property>
            <property name="cursorPosition">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string> Alt </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Alt">
            <property name="text">
             <string>000.0</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>WayPoints</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QComboBox" name="cbLight"/>
          </item>
          <item>
           <widget class="QPushButton" name="btnAdd">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearAllPointButton">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="takeoffButton">
            <property name="text">
             <string>TakeOff</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="waypointAlt"/>
          </item>
          <item>
           <widget class="QPushButton" name="goButton">
            <property name="text">
             <string>Go</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Car detection</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="detect_carButton">
            <property name="text">
             <string>detect_car</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="track_carButton">
            <property name="text">
             <string>track_car</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="do_nothing">
            <property name="text">
             <string>wait</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>     number:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditnumber"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>qwebengineview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
